tests/integration/test_storage.py

```diff
@@ -17,10 +18,10 @@
 
 logger = logging.getLogger(__name__)
 
-MAX_RETRIES = 20
 INSUFFICIENT_SIZE_WARNING = "<10% free space on pgdata volume."
 
 
+@markers.amd64_only
```

Comment:
Hostpath  has no size limitations, so we are filling the host disk. We should most probably start using real volumes in the tests (Microceph? https://microk8s.io/docs/how-to-ceph)